### ISSUE TITLE
Can't run the script on falun.zip

### ISSUE THREAD

**Issue by matthiasfeist** [atorger/nvdb2osm#11]
Hi.

I though I'd give your script a shot to map on the Falun data, but I get an error thrown and I'm not sure if it is me doing anything wrong, or if there is a problem with the data from Falun:

``` python3 ./nvdb2osm.py falun.zip falun.osm
2021-02-24 14:30:12,881 - waydb - WARNING - extend snaps typically means that there are gaps in the data source's geometry
2021-02-24 14:30:23,796 - waydb - WARNING - RLID 13649:1854 is not in reference geometry, inserting it.
Traceback (most recent call last):
  File "/Users/matthiasf/src/nvdb2osm/waydb.py", line 950, in _adapt_way_into_reference_geometry
    ref_p = next(ref_it)
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./nvdb2osm.py", line 309, in <module>
    main()
  File "./nvdb2osm.py", line 236, in main
    insert_rlid_elements(way_db, ways, name, debug_ways=debug_ways)
  File "./nvdb2osm.py", line 102, in insert_rlid_elements
    way_db.insert_rlid_way(way, data_src_name, debug_ways)
  File "/Users/matthiasf/src/nvdb2osm/waydb.py", line 708, in insert_rlid_way
    _, ways = self._adapt_way_into_reference_geometry(way, data_src_name)
  File "/Users/matthiasf/src/nvdb2osm/waydb.py", line 952, in _adapt_way_into_reference_geometry
    raise RuntimeError("Could not find start %s of way %s in reference geometry (does it extend reference geometry?)" % (latlon_str(way.way[0]), way.rlid)) from stop_iteration
RuntimeError: Could not find start <lat:60.87480924834481 lon:15.613082402002087> of way 1000:99311 in reference geometry (does it extend reference geometry?)
```

**Comment by atorger** [comment]
I'll have a look this evening. The NVDB data has some quirks, the script corrects for most of them but every now and then when I run a new kommun I need to add yet another fix/workaround. I haven't tried Falun before.

**Comment by atorger** [comment]
I just committed a patch so now it should work (works for me, tested on Falun), turns out that the maximum bridging gap was 1.0 meters, which has been sufficient so far, but as Falun has one place with 2.4 meters, I now increased the limit to 2.5. The risk of false bridging is small despite this increase as this type of bridging only goes to matching RLIDs. (The script does other types of gap bridging too).
